src/main/java/org/opensearch/ad/model/AnomalyDetector.java

```diff
@@ -95,6 +94,7 @@
     public static final String USER_FIELD = "user";
     public static final String DETECTOR_TYPE_FIELD = "detector_type";
     public static final String RESULT_INDEX_FIELD = "result_index";
+    public static final String MODEL_VALIDATION_ISSUE = "aggregation_issue";
```

Comment:
Does model validation issue equal aggregation issue?

Reply:
At first it only really happened when aggregation failed for some reason (could be a change in core) but now I also use this for timeout exception. However based on a different comment you made, I will make a different exception for timeout issue and pass that specifically to frontend.

Reply:
If it is only about aggregation, maybe change the variable name to be more specific?